doc/source/tutorials/storage.rst

```diff
@@ -0,0 +1,32 @@
+Platforms storage
+=================
+
+To define a platform the user needs to provide a folder with the following structure:
+
+.. code-block:: bash
+
+    my_platform/
+        platform.py
+        parameters.json
+
+where ``platform.py`` contains instruments information and ``parameters.json`` includes calibration parameters.
+
+Setting up the environment
+--------------------------
+
+After defining the platform, we must instruct ``qibolab`` of the location of the platform(s).
+We need to define the path that contains platform folders.
+This can be done using an environment variable:
+for Unix based systems:
+
+.. code-block:: bash
+
+    export QIBOLAB_PLATFORMS=<path-platform-folders>
+
+for Windows:
+
+.. code-block:: bash
+
+    $env:QIBOLAB_PLATFORMS="<path-to-platform-folders>"
+
+To avoid having to repeat this export command for every session, this line can be added to the ``.bashrc`` file (or alternatives such as ``.zshrc``).
```

Comment:
I know that this is still a draft but it would be good to mention how to handle multiple platform paths to close https://github.com/qiboteam/qibolab/issues/1161.

Reply:
Thanks for the remainder. 

For the time being, this is just a cut-and-paste of previous paragraphs, not to forget them. 

But yes, the goal is definitely to expand way more, to bring the docs up-to-date.

Suggestions are more than welcome, and I'll maintain a to-do list in the OP as usual.

Reply:
Now in https://github.com/qiboteam/qibolab/issues/1236